mlir/test/CAPI/ir.c

```diff
@@ -397,23 +397,15 @@ static void printFirstOfEach(MlirContext ctx, MlirOperation operation) {
   // CHECK: Terminator: func.return
 
   // Get the attribute by index.
-  MlirNamedAttribute namedAttr0 = mlirOperationGetAttribute(operation, 0);
-  fprintf(stderr, "Get attr 0: ");
-  mlirAttributePrint(namedAttr0.attribute, printToStderr, NULL);
+  MlirAttribute valueAttr0 = mlirOperationGetInherentAttributeByName(operation, mlirStringRefCreateFromCString("value"));
```

Comment:
Nit: 80 cols

Reply:
It used to be checked by `arc` for me :(